hooks/useSubmission.tsx

```
@@ -44,8 +45,9 @@ class SubmissionController {
   readonly submission_file_comments: TableController<"submission_file_comments">;
   readonly submission_artifact_comments: TableController<"submission_artifact_comments">;
   readonly submission_reviews: TableController<"submission_reviews">;
+  readonly submission_regrade_request_comments: TableController<"submission_regrade_request_comments">;
 
-  readonly readyPromise: Promise<[void, void, void, void]>;
+  readonly readyPromise: Promise<[void, void, void, void, void]>;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)